src/Components/Canvas/Canvas.tsx

```diff
@@ -138,68 +139,64 @@ function sketch(p) {
 }
 
 export default function Canvas(props) {
-    // const defaultMouseEvent = hawkeyeMouseEvent: {
-    //     mousePressed: false,
-    //     mouseX: 0,
-    //     mouseY: 0
-    // };
+    const defaultMouseEvent : HawkeyeMouseEvent = {
+        mousePressed: false,
+        mouseX: 0,
+        mouseY: 0
+    };
 
-    // const [hawkeyeMouseEvent, setHawkeyeMouseEvent] = useState(defaultMouseEvent);
+    const [hawkeyeMouseEvent, setHawkeyeMouseEvent] = useState(defaultMouseEvent);
     const [xpos, setXpos] = useState(0);
     const [ypos, setYpos] = useState(0);
 
     function gridClickedHandler(id) {
-        // let id = e.target.id;
         let element = document.getElementById(id);
-        let xpos = element.offsetTop + element.offsetHeight / 2;
-        let ypos = element.offsetLeft + element.offsetWidth / 2;
+        let xpos = element.offsetLeft - 10;
+        let ypos = element.offsetTop;
```

Comment:
Why'd you remove `element.offsetHeight / 2` and `element.offsetWidth / 2`? Do you not want the mouse position to be the center of the element?